pkg/virt-controller/services/template_test.go

```diff
@@ -1478,15 +1478,6 @@ var _ = Describe("Template", func() {
 				mem := resource.MustParse("64M")
 				Expect(pod.Spec.Containers[1].Resources.Limits.Memory().Cmp(mem)).To(BeZero())
 				Expect(pod.Spec.Containers[1].Resources.Limits.Cpu().Cmp(cpu)).To(BeZero())
-
-				found := false
-				caps := pod.Spec.Containers[0].SecurityContext.Capabilities
-				for _, cap := range caps.Add {
-					if cap == CAP_SYS_NICE {
-						found = true
-					}
-				}
-				Expect(found).To(BeTrue(), "Expected compute container to be granted SYS_NICE capability")
```

Comment:
I agree, and this VMI is considered root since it has no non-root annotation and no status.
I verified and the test still passes with this test back in. I guess I'll re-add it in the next update.

Reply:
Re-added in new commit